Makefile

```diff
@@ -8,6 +8,9 @@ target test-chrome: url=http://$(site).docker.test
 
 include .env
 
+NPM_PATH := ./node_modules/.bin
+export PATH := $(NPM_PATH):$(PATH)
+
```

Comment:
A better way is to add the `api-docs` command as a `script` in `package.json`. That way, these lines here can be removed.

Reply:
I don't see how this is better. We already use the above construction in several other places, so for the sake of consistency I would like to keep it this way.